view/adminhtml/templates/transaction_sync.phtml

```diff
@@ -70,7 +70,7 @@
                 $('#transaction-sync-log').find('> pre').html('Syncing transactions to TaxJar, please wait...');
 
                 $.ajax({
-                    url: '<?php echo $block->escapeUrl($block->getStoreUrl('taxjar/transaction/backfill')) ?>',
+                    url: '<?php echo $block->escapeUrl($block->getStoreUrl('taxjar/transaction/backfill', $block->getRequest()->getParams())) ?>',
```

Comment:
@ryanreeves-taxjar On Magento 2.2 this addition returns back the following error in the AJAX response when backfilling from a specific store view:

```
{"error":true,"message":"You entered an invalid Secret Key. Please refresh the page."}
```

I'm seeing `/section/tax/store/7/` appended to the URL before the query params: `/admin/taxjar/transaction/backfill/key/:key/section/tax/store/7/?from_date=04%2F1%2F2020&to_date=&isAjax=true`

Can we pass the params in the `data` object similar to `from_date` and `to_date`?